hack/nudging/container_digest.sh

```diff
@@ -1,8 +1,8 @@
 # Do not remove empty lines, they are there to reduce conflicts
-export OPERATOR_IMAGE_PULLSPEC='registry.redhat.io/network-observability/network-observability-rhel9-operator@sha256:304440234657a05d3fde85a780f23b29c80fcb7089b38c1b86b24dd799665c54'
+export OPERATOR_IMAGE_PULLSPEC='registry.redhat.io/network-observability/network-observability-rhel9-operator@sha256:e059192784283385687011b9fa4bb87094783d4530aa66b79168777666a6e2a0'
 #
-export EBPF_IMAGE_PULLSPEC='registry.redhat.io/network-observability/network-observability-ebpf-agent-rhel9@sha256:67191ccd3c6631e7e7dd56789a4e894db1fc8a5de0e653f0bee6e6230a3affaf'
+export EBPF_IMAGE_PULLSPEC='registry.redhat.io/network-observability/network-observability-ebpf-agent-rhel9@sha256:96b3765ea74f74f2a4043767dc9e1ff0bf6b47ced011999fe91e27529bce6133'
 #
-export FLP_IMAGE_PULLSPEC='registry.redhat.io/network-observability/network-observability-flowlogs-pipeline-rhel9@sha256:460b0cceae39b13d3d9b21b0e8364342866435afeff9db59f085d6571e236ef6'
+export FLP_IMAGE_PULLSPEC='registry.redhat.io/network-observability/network-observability-flowlogs-pipeline-rhel9@sha256:c6c194d9f631f0b5233b1edfc52381d55fa0d64a84d9d354ec85e774be4f0ef6'
 #
-export CONSOLE_IMAGE_PULLSPEC='registry.redhat.io/network-observability/network-observability-console-plugin-rhel9@sha256:95b58585c1d9db9ad3b3e72d9263da91328a67a3cf2192b44bbee0c89d407687'
+export CONSOLE_IMAGE_PULLSPEC='registry.redhat.io/network-observability/network-observability-console-plugin-rhel9@sha256:7fa0b4ded5e30668be09fcd953a9912d9bc31d610419a281583d7f916694a49f'
```

Comment:
@OlivierCazade how do you pick those digest references? for instance I see these builds are not yet completed and they seem to be for unmerged PRs/commits.

Reply:
I picked them from the last release snapshot on the konflux snapshot.

Are you sure they correspond to unmerged builds ? Only build from main branch should trigger new snapshot.

Reply:
I think @OlivierCazade is correct.
This konflux UI view is a bit misleading, what is shows in the left column is the last image built from the HEAD commit (ie. things that are merged) and on the right column is the last build, including unmerged builds. So the image on the left doesn't necessarily correspond to the build on the right.
You can verify that because PR image tags a prefixed with "on-pr-", like this one:
```
quay.io/redhat-user-workloads/ocp-network-observab-tenant/netobserv-operator/network-observability-operator-bundle:on-pr-867f4a431c4c231acc5c68752fcd4c1a7ff3edf0
```

Reply:
Moreover the fact that the images come from the release snapshot should be sufficient to be trusted, there's no way release snapshot could contained unmerged stuff, unless the whole thing being totally screwed up :-)

Reply:
> This konflux UI view is a bit misleading, what it shows in the left column is the last image built from the HEAD commit (ie. things that are merged) and on the right column is the last build, including unmerged builds. 

ah, thanks!